zipline/data/us_equity_pricing.py

```diff
@@ -655,8 +659,11 @@ def calc_dividend_ratios(self, dividends):
             try:
                 prev_close = daily_bar_reader.spot_price(
                     sid, div_adj_date, 'close')
-                ratio = 1.0 - amount / (prev_close)
-                ratios[i] = ratio
+                if prev_close != 0.0:
+                    ratio = 1.0 - amount / (prev_close)
```

Comment:
did you mean to put parens around `amount / prev_close` ?  (instead of around `prev_close`)